tests/Custom/CapabilityTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Mink\WebdriverClassicDriver\Tests\Custom;
+
+use Mink\WebdriverClassicDriver\WebdriverClassicDriver;
+
+class CapabilityTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @param array<string, mixed> $desiredCapabilities
+     * @param array<string, mixed> $expectedCapabilities
```

Comment:
Is this correct behavior to specify only some of the method parameters in the PhpDoc?

Reply:
Yes - PHPDoc is to augment or narrow existing information. E.g. in this case adding a line for `string $browserName` is quite redundant (unless we want to add something special to that parameter - such as a description or `non-empty-string` instead of `string`..).

PHP-CS-Fixer even has a rule/fixer for this: [no_superfluous_phpdoc_tags](https://cs.symfony.com/doc/rules/phpdoc/no_superfluous_phpdoc_tags.html).

Reply:
OK.